app/components-react/highlighter/StreamCard.tsx

```diff
@@ -49,12 +52,77 @@ export default function StreamCard({
     return <></>;
   }
 
+  function shareFeedback() {
+    remote.shell.openExternal(
+      'https://support.streamlabs.com/hc/en-us/requests/new?ticket_form_id=31967205905051',
+    );
+  }
+
   function showStreamClips() {
     if (stream?.state.type !== EAiDetectionState.IN_PROGRESS) {
       emitSetView({ view: EHighlighterView.CLIPS, id: stream?.id });
     }
   }
 
+  if (stream.state.type === EAiDetectionState.FINISHED && clips.length === 0) {
+    return (
+      <div className={styles.streamCard}>
+        {' '}
+        <Button
+          size="large"
+          className={styles.deleteButton}
+          onClick={e => {
+            emitRemoveStream();
+            e.stopPropagation();
+          }}
+          style={{ backgroundColor: '#00000040', border: 'none', position: 'absolute' }}
+        >
+          <i className="icon-trash" />
+        </Button>
+        <div className={styles.requirements}>
+          <div style={{ display: 'flex', flexDirection: 'column', width: '280px' }}>
+            <h2>{$t('No clips found')}</h2>
+            <p style={{ marginBottom: '8px' }}>
+              {$t('Please make sure all the requirements are met:')}
+            </p>
+            <p>{$t('• Game is supported (Currently Fortnite only)')}</p>
+            <p>{$t('• Game language is English')}</p>
+            <p>{$t('• Map and Stats area is fully visible')}</p>
+            <p>{$t('• Game in fullscreen in your stream')}</p>
+            <p>{$t('• Game mode is supported (Battle Royale, Reload, Zero Build, OG)')}</p>
```

Comment:
i think here we would probably prefer to have a `<ul>` tag here just to keep translated strings to only what is required of them